module.exports = "restoring_app=Memuat...\ninitializing_odata=Memulakan Data Luar Talian\nqr_code_message_success=Tetapan sambungan baharu berjaya dimuatkan.\nqr_code_message_fail=Gagal memuat tetapan sambungan baharu.\nunable_to_load_new_connection_setttings=Tidak boleh memuat tetapan sambungan baharu. Sila log keluar/tetapkan semula sebelum memuat tetapan sambungan baharu.\noverride_configuration_settings=Tetapan sambungan baharu berjaya dimuatkan dari fail konfigurasi pembatalan.\nupdate_fail_roll_back=Pengemaskinian aplikasi gagal, akan gulung balik ke takrifan metadata sebelumnya\\nSebab:\nclose_drawer=Tutup laci\ndismiss=GUGURKAN\nback=Kembali";
